source/Makefile.vars

```diff
@@ -16,6 +16,10 @@ CC = mpiicpc
 # GPU = OFF  #We do not support GPU yet
 # OFF:  do not use GPU
 # CUDA: use CUDA
+OPENMP = OFF 
+# the default is not to use OPENMP to accelerate. 
```

Comment:
ABACUS by default uses OpenMP. Do you mean the OpenMP support for ELPA? I suggest pinning this option to true.

Reply:
In the image, the ELPA is compiled with -openmp option turned on, thereby generating a library file named 'elpa_openmp.a'. However, in most cases, we only compile ELPA without the openmp option, generating 'elpa.a' library. This is why I put a OFF here.